.github/workflows/release.yml

```diff
@@ -9,11 +9,19 @@ on:
       - 'V*.*.*'
 jobs:
   release:
-    runs-on: ubuntu-latest
+    runs-on: 
+      - self-hosted
```

Comment:
Why do we want this to run on self hosted severs?

Reply:
fixed